src/components/SearchModal/Manage.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useCallback, useState } from 'react'
+import { RowBetween } from '../Row'
+import { Box, Flex, Text } from 'rebass'
+import styled from 'styled-components'
+import { Token } from 'dxswap-sdk'
+import { ManageLists } from './ManageLists'
+import ManageTokens from './ManageTokens'
+import { TokenList } from '@uniswap/token-lists'
+import { CurrencyModalView } from './CurrencySearchModal'
+import { CloseIcon } from '../../theme'
+import { useMeasure } from 'react-use'
+import { animated, useSpring } from '@react-spring/web'
+import { GoBackIcon } from './styleds'
+
+const Wrapper = styled(Flex)`
+  width: 100%;
+  background-color: ${({ theme }) => theme.bg1And2};
+  flex-direction: column;
+`
+
+const ToggleWrapper = styled(RowBetween)`
+  position: relative;
+  background-color: ${({ theme }) => theme.bg2};
+  border-radius: 12px;
+  padding: 6px;
+`
+
+const TabContainer = styled.div`
+  display: flex;
+  overflow: hidden;
+`
+
+const Slide = styled.div`
+  width: 100%;
+  flex-shrink: 0;
+`
+const AnimatedSlide = animated(Slide)
+
+const ToggleOption = styled.div<{ active: boolean }>`
```

Comment:
I would replace div with a button. It will be more semantic. You will need to reset some button styles, because button has some default styles.

```suggestion
const ToggleOption = styled.button<{ active: boolean }>`
```

Reply:
Fixed here edc4a79f94bf87b80f26cd1d65398ae99d8506bf